File: frontend/src/app/models/institucion.ts
export interface Institucion {
  id: number;
  codigo: string;
  nombre: string;
  municipio: string;
  activa: boolean;
}

export interface PagedResult<T> {
  items: T[];
  total: number;
  pagina: number;
  tamano: number;
}

File: backend/GestionPae.Api/Models/Institucion.cs
namespace GestionPae.Api.Models;

public class Institucion
{
    public int Id { get; set; }
    public string Codigo { get; set; } = default!;
    public string Nombre { get; set; } = default!;
    public string Municipio { get; set; } = default!;
    public bool Activa { get; set; }
}
